homeassistant/components/tuya/const.py

```diff
@@ -773,3 +775,238 @@ class Country:
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 120,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "jump",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 240,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "jump",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+        ],
+    },
+    "Beautiful": {
+        "scene_num": 8,
+        "scene_units": [
+            {
+                "bright": 0,
+                "h": 0,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 120,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 240,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 61,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 174,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+            {
+                "bright": 0,
+                "h": 275,
+                "s": 1000,
+                "temperature": 0,
+                "unit_change_mode": "gradient",
+                "unit_gradient_duration": 70,
+                "unit_switch_duration": 70,
+                "v": 1000,
+            },
+        ],
+    },
+}
+
+WHITE_SUPPORTED_SCENES = ("Night", "Read", "Meeting", "Leisure")
+COLOR_SUPPORTED_SCENES = (
+    "Night",
+    "Read",
+    "Meeting",
+    "Leisure",
+    "Soft",
+    "Rainbow",
+    "Shine",
+    "Beautiful",
```

Comment:
hey @frenck , did you get any feedback from tuya?

where this is I take it that it they need to provide a list of "scenes" from an API of some sort for this feature to work as coded? otherwise a some front end way to configure these "scenes" would need to be created?

Reply:
> did you get any feedback from tuya?

Sorry forgot about it.